apps/hpc-ftsadmin/src/app/utils/parse-filters.ts

```diff
@@ -126,7 +126,8 @@ const parseInInitialValues = <T extends Filters>(
   initialValues: T
 ) => {
   for (const key in initialValues) {
-    filters[key] = filters[key] ? filters[key] : initialValues[key];
+    filters[key] =
+      filters[key] !== undefined ? filters[key] : initialValues[key];
```

Comment:
This is my code too, but I think it could be improved if:
```suggestion
filters[key] ?? initialValues[key];
```
If I'm not mistaken, it should work